### PR TITLE
Check-in refresh plugin configs

### PR DESCRIPTION
**Pull request by 0xmichalis**
Depends on https://github.com/kubernetes/test-infra/pull/5075

/hold

### CODE REVIEW

File: cluster/ci/config/prow/prow_images.yaml [hunk 1]
```diff
@@ -37,8 +37,7 @@ objects:
             mkdir -p /static && \
             cp -r cmd/deck/static/* /static && \
             CGO_ENABLED=0 go install -a -installsuffix cgo -ldflags '-w' ./cmd/... && \
-            cd ../experiment/cherrypicker && \
-            CGO_ENABLED=0 go install && \
+            CGO_ENABLED=0 go install ../experiment/... && \
```
Comment (stevekuznetsov) [inline cluster/ci/config/prow/prow_images.yaml]: This seems like a bad idea

Comment (kargakis) [reply cluster/ci/config/prow/prow_images.yaml]: Updated to compile the plugins we want.

**Comment by stevekuznetsov** [comment]
@kargakis i meant -- this may be a good time to break it out into it's own build and own base image, what is the pros of adding it into the mega binaries image tag?

**Comment by kargakis** [comment]
> @kargakis i meant -- this may be a good time to break it out into it's own build and own base image, what is the pros of adding it into the mega binaries image tag?

SGTM - added a BC for the plugins

**Pull request merged**